crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -90,6 +90,23 @@ def addons
       end
     end
 
+    def repocheck(options = {})
+      platform = options.fetch(:platform)
+      arch = options.fetch(:arch)
+
+      repos = {}
+      wanted_repos = ["cloud", "sles12-sp2-pool", "sles12-sp2-updates"]
```

Comment:
Ah, and this is even the master code. Here we don't have any idea what could be the needed repo for the 7-8 upgrade...